### DISCUSSION TITLE
[@types/jest] missing expect.closeTo

### DISCUSSION THREAD

**Discussion by mmmmmrob** [DefinitelyTyped/DefinitelyTyped#60613] [Issues with a @types package] [answered]
I have jest installed as `"jest": "^27.2.1"` and @types/jest installed as `"@types/jest": "^27.0.2"`.

I am getting a typescript error `TS2339: Property 'closeTo' does not exist on type 'Expect'.`

for the line `expect(actualData).toMatchObject([[expect.closeTo(3.01)]])`

As far as I can see `expect.closeTo` is not defined in @types/jest.

Is this a missing declaration or am I doing something wrong?

[a comment by an automated account is not shown]

**Comment by domdomegg** [comment, marked as answer]
Thanks for raising this! Yes, this is missing from the types. I'll work on a PR to add them now.

**Reply by domdomegg** [reply]
I've raised https://github.com/DefinitelyTyped/DefinitelyTyped/pull/60631

**Reply by mmmmmrob** [reply]
Thank you.
